.github/workflows/ci-sdk-testkit.yml

```diff
@@ -0,0 +1,66 @@
+name: Continuous Integration Delta SDK Testkit
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-testkit/**'
+      - 'build.sbt'
+      - 'project/**'
+  pull_request:
+    paths:
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-testkit/**'
+      - 'build.sbt'
+      - 'project/**'
+jobs:
+  review:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - name: Cache Coursier
```

Comment:
We could possibly replace these across all jobs to use coursier cache action: https://github.com/marketplace/actions/coursier-cache-action